src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -924,6 +925,94 @@ public bool IsMouseButtonReleased(MouseButton button)
             return !_mouseState.IsButtonDown(button) && LastMouseState.IsButtonDown(button);
         }
 
+        /// <inheritdoc />
+        public bool GetCurrentMonitorScale(out float horizontalScale, out float verticalScale)
+        {
+            if (CurrentMonitor.Pointer != IntPtr.Zero)
```

Comment:
Can the monitor ever be null? you should probably do a bit of research into this, but i dont believe so, try checking the constructor